test/unit/cwe_checker_unit_tests.ml

```diff
@@ -1,26 +1,89 @@
 open Bap.Std
 open Core_kernel
 
+include Self()
+
+let cmdline_params = [
+  ("partial", "Comma separated list defining which tests should be executed with the current test file. e.g. MemRegion,TypeInference,CWE476,...")
+]
+
+
+let input_test_map = Map.of_alist_exn (module String) [
+  "MemRegion", "Mem_region_tests";
+  "TypeInference", "Type_inference_tests";
+  "CWE476", "CWE_476_tests";
+  "CWE560", "CWE_560_tests";
+  "AddrTrans", "Address_translation_tests";
+  "Cconv", "Cconv_tests"
+]
```

Comment:
Seems unneccessary. Can you get rid of it by just removing `_test` from the test names?